src/firmware/posix/CMakeLists.txt

```diff
@@ -140,4 +140,12 @@ install(TARGETS px4 DESTINATION ${CMAKE_INSTALL_BINDIR})
 install(DIRECTORY ${PROJECT_SOURCE_DIR}/ROMFS DESTINATION ${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME})
 install(DIRECTORY ${PROJECT_SOURCE_DIR}/posix-configs DESTINATION ${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME})
 
+add_custom_target(strip
+			DEPENDS px4.stripped
+)
+
+add_custom_command(OUTPUT px4.stripped
+			COMMAND ${STRIP_TOOL} -R .comment -R .gnu.version $<TARGET_FILE:px4>
```

Comment:
Do we need the upload custom command to depend on the stripped output instead? https://github.com/nicolaerosia/Firmware/blob/f4ba9c5acb3a0efec823e398e76bbbf976013372/src/firmware/posix/CMakeLists.txt#L103